doc/api/dns.md

```diff
@@ -204,8 +204,9 @@ changes:
 * `options` {integer | Object}
   * `family` {integer|string} The record family. Must be `4`, `6`, or `0`. For
     backward compatibility reasons,`'IPv4'` and `'IPv6'` are interpreted as `4`
-    and `6` respectively. The value `0` indicates that IPv4 and IPv6 addresses
-    are both returned. **Default:** `0`.
+    and `6` respectively. The value `0` indicates that either IPv4 or IPv6
```

Comment:
```suggestion
    and `6` respectively. The value `0` indicates that either a IPv4 or IPv6
```

Reply:
(not a native speaker, so I can certainly be wrong, but would you say "an IPv4 address" rather than "a IPv4 address"?)